executor/show_test.go

```diff
@@ -951,9 +951,9 @@ func (s *testSuite5) TestShowBuiltin(c *C) {
 	res := tk.MustQuery("show builtins;")
 	c.Assert(res, NotNil)
 	rows := res.Rows()
-	c.Assert(268, Equals, len(rows))
+	c.Assert(269, Equals, len(rows))
```

Comment:
`IsTruthWithNull` should not shown in `show builtins`

Reply:
Addressed.